pkg/validationfile/testdata/invalid_caveated_rel_syntax.yaml

```diff
@@ -0,0 +1,19 @@
+---
+schema: >-
+  definition user {}
+
+  caveat some_caveat(somecondition int) {
+    somecondition == 42
+  }
+
+  definition resource {
+      relation reader: user with some_caveat
+  }
+relationships: >-
+  resource:first#reader@user:tom[some_caveat]
+
+  resource:first#reader@user:sarah[some_caveat{"somecondition": 42}]
```

Comment:
let's add a test with this rel duplicated, but one with and another without context, so we can demonstrate the dupe is ignored in order they are found.